misk-jooq/src/main/kotlin/misk/jooq/JooqModule.kt

```diff
@@ -10,96 +11,128 @@ import misk.inject.toKey
 import misk.jdbc.DataSourceClusterConfig
 import misk.jdbc.DataSourceConfig
 import misk.jdbc.DataSourceService
-import misk.jdbc.DataSourceType
 import misk.jdbc.DatabasePool
 import misk.jdbc.JdbcModule
 import misk.jdbc.RealDatabasePool
-import misk.jooq.listeners.AvoidUsingSelectStarListener
-import misk.jooq.listeners.JooqSQLLogger
-import misk.jooq.listeners.JooqTimestampRecordListener
+import misk.jooq.config.CachedConfigurationFactory
+import misk.jooq.config.ConfigurationFactory
 import misk.jooq.listeners.JooqTimestampRecordListenerOptions
 import org.jooq.Configuration
-import org.jooq.DSLContext
-import org.jooq.SQLDialect
-import org.jooq.conf.MappedSchema
-import org.jooq.conf.RenderMapping
-import org.jooq.conf.Settings
-import org.jooq.impl.DSL
-import org.jooq.impl.DefaultExecuteListenerProvider
-import org.jooq.impl.DefaultTransactionProvider
 import java.time.Clock
 import kotlin.reflect.KClass
 
 class JooqModule @JvmOverloads constructor(
-  private val qualifier: KClass<out Annotation>,
-  private val dataSourceClusterConfig: DataSourceClusterConfig,
-  private val jooqCodeGenSchemaName: String,
-  private val databasePool: DatabasePool = RealDatabasePool,
-  private val readerQualifier: KClass<out Annotation>? = null,
-  private val jooqTimestampRecordListenerOptions: JooqTimestampRecordListenerOptions =
-    JooqTimestampRecordListenerOptions(install = false),
-  private val installHealthChecks: Boolean = true,
-  private val installSchemaMigrator: Boolean = true,
-  private val jooqConfigExtension: Configuration.() -> Unit = {},
+    private val qualifier: KClass<out Annotation>,
```

Comment:
Can you reverse the formatting differences? Seems to have moved from 2 spaces to 4.